ui/raidboss/data/06-ew/raid/p12s.txt

```diff
@@ -4,3 +4,169 @@ hideall "--Reset--"
+324.4 "--targetable--"
+333.2 "Theos's Ultima" sync / 1[56]:[^:]*:Athena:82FA:/
+352.6 "Superchain Theory IIA" sync / 1[56]:[^:]*:Athena:86FA:/
+365.7 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/ window 1.5,0.5
+368.3 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/ window 1.5,0.5
+370.7 "Trinity of Souls" sync / 1[56]:[^:]*:Athena:82E[1-9A-C]:/ window 1.5,0.5
+374.0 "Superchain Coil/Burst" sync / 1[56]:[^:]*:Athena:82D[BC]:/ window 1.5,3
+380.2 "Apo/Peridialogos" sync / 1[56]:[^:]*:Athena:830[01]:/
+381.2 "Dialogos" sync / 1[56]:[^:]*:Athena:8302:/
+387.9 "On the Soul" sync / 1[56]:[^:]*:Athena:8304:/
+395.1 "--sync--" sync / 1[56]:[^:]*:Athena:8315:/
+402.5 "Superchain Theory IIB" sync / 1[56]:[^:]*:Athena:86FB:/
+408.6 "Paradeigma" sync / 1[56]:[^:]*:Athena:82ED:/
+415.6 "Superchain Coil/Burst" sync / 1[56]:[^:]*:Athena:82D[BC]:/ window 3,0.5
+418.6 "Parthenos" sync / 1[56]:[^:]*:Athena:8303:/
+422.5 "Superchain Coil/Burst" sync / 1[56]:[^:]*:Athena:82D[BC]:/ window 3,0.5
+424.9 "Ray of Light" sync / 1[56]:[^:]*:Anthropos:82EE:/
+431.5 "Superchain Coil/Burst" sync / 1[56]:[^:]*:Athena:82D[BC]:/ window 3,0.5
+434.3 "Unnatural Enchainment" sync / 1[56]:[^:]*:Athena:82DF:/
+442.4 "On the Soul" sync / 1[56]:[^:]*:Athena:8304:/
+450.5 "On the Soul" sync / 1[56]:[^:]*:Athena:8304:/
+459.0 "--untargetable--" sync / 14:[^:]*:Athena:8556:/
+466.0 "Theos's Ultima (enrage)" sync / 1[56]:[^:]*:Athena:8556:/
+
+# P2 Pallas Athena -p 8682:1012.1 -ii 831A 831B 879F 832C 87A0 8334 8325 8323 833D 8792 833E 86FD 831F 8683 8320 8322 8340 8328
+1007.1 "--sync--" sync / 14:[^:]*:Pallas Athena:8682:/ window 1100,10
+1012.1 "Ultima" sync / 1[56]:[^:]*:Pallas Athena:8682:/
+1024.5 "Palladian Grasp" sync / 1[56]:[^:]*:Pallas Athena:831[CD]:/
+1027.7 "Palladian Grasp" sync / 1[56]:[^:]*:Pallas Athena:831[CD]:/
+1040.8 "Gaiaochos" sync / 1[56]:[^:]*:Pallas Athena:8326:/
+1051.9 "Summon Darkness" sync / 1[56]:[^:]*:Pallas Athena:832F:/
+1063.8 "Ultima Ray" sync / 1[56]:[^:]*:Hemitheos:8330:/
+1065.1 "Missing Link" sync / 1[56]:[^:]*:Pallas Athena:8586:/
+1074.1 "Demi Parhelion" sync / 1[56]:[^:]*:Pallas Athena:8327:/
+1084.2 "Geocentrism" sync / 1[56]:[^:]*:Pallas Athena:832[AB9]:/
+1085.2 "Divine Excoriation" sync / 1[56]:[^:]*:Pallas Athena:832E:/
+1094.4 "Ultima" sync / 1[56]:[^:]*:Pallas Athena:86F6:/
+1108.6 "The Classical Concepts" sync / 1[56]:[^:]*:Pallas Athena:8331:/
+1130.0 "Implode" #sync / 1[56]:[^:]*:Concept of (Earth|Water|Fire):8333:/
+1133.4 "Palladian Ray" sync / 1[56]:[^:]*:Pallas Athena:8324:/
+1143.8 "Ultima" sync / 1[56]:[^:]*:Pallas Athena:8682:/
+1153.0 "Crush Helm" sync / 1[56]:[^:]*:Pallas Athena:8317:/
+1153.8 "Crush Helm 1" sync / 1[56]:[^:]*:Pallas Athena:8318:/ window 0.5,0.5
+1154.9 "Crush Helm 2" sync / 1[56]:[^:]*:Pallas Athena:8318:/ window 0.5,0.5
+1156.0 "Crush Helm 3" sync / 1[56]:[^:]*:Pallas Athena:8318:/ window 0.5,0.5
+1157.1 "Crush Helm 4" sync / 1[56]:[^:]*:Pallas Athena:8318:/ window 0.5,0.5
```

Comment:
```suggestion
1153.8 "Crush Helm 1" #sync / 1[56]:[^:]*:Pallas Athena:8318:/
1154.9 "Crush Helm 2" #sync / 1[56]:[^:]*:Pallas Athena:8318:/
1156.0 "Crush Helm 3" #sync / 1[56]:[^:]*:Pallas Athena:8318:/
1157.1 "Crush Helm 4" #sync / 1[56]:[^:]*:Pallas Athena:8318:/
```

These are too close in time, so would prefer to just comment out the syncs.  It's easy for stuff like this to throw off the timeline.

Reply:
I didn't have any issues when testing it, and my assumption at the time was that since the timing is tight (healers need to Esuna right after the fourth hit or tanks go splat) that it would be important to keep in. But, in hindsight, if someone has lag or packet loss it could likely mess it up, yeah.